src/components/Footer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInstagram } from '@fortawesome/free-brands-svg-icons';
+import { faFacebookSquare } from '@fortawesome/free-brands-svg-icons';
+import { faWhatsapp } from '@fortawesome/free-brands-svg-icons';
```

Comment:
@Felipe-Cadavid A better and simple way to import multiple icons from the same file, is adding only one line and import const separated with comma.

For example:

`import { faInstagram, faFacebookSquare, faWhatsapp } from '@fortawesome/free-brands-svg-icons';`

Reply:
@leoneloliveros thank you so much for your review, it's already resolved